pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -303,9 +303,9 @@ def test_join_with_period_index(self, join_type):
             c_idx_type='p', r_idx_type='dt')
         s = df.iloc[:5, 0]
 
```

Comment:
can you make this a separate test

Reply:
not sure what you mean by a separate test. The existing test is no longer valid, the changes I made here reflect how the modifications in this branch would affect this test. Also this answers your question from above?

https://github.com/pandas-dev/pandas/pull/23538/files#r238115422